### ISSUE TITLE
Build failure: cemu

### ISSUE THREAD

**Issue by joygnu** [NixOS/nixpkgs#354303]
### Steps To Reproduce

1. build *cemu*

### Build log

```
       > /build/source/src/gui/debugger/SymbolWindow.cpp:12: note: mismatching definition
       >    12 |         ColumnModule,
       >       |
       > /build/source/src/Cafe/HW/Latte/Core/LatteTexture.cpp: In function 'LatteTexture_InitSliceAndMipInfo':
       > /build/source/src/Cafe/HW/Latte/Core/LatteTexture.cpp:193: warning: argument 1 value '18446744073709551615' exceeds maximum object size 9223372036854775807 [-Walloc-size-larger-than=]
       >   193 |         texture->sliceMipInfo = new LatteTextureSliceMipInfo[mipSliceCount]();
       >       |
       > /nix/store/v7dbfh5n7az2lcap0z1cv4jq0bikya8p-gcc-13.3.0/include/c++/13.3.0/new:128:26: note: in a call to allocation function 'operator new []' declared here
       >   128 | _GLIBCXX_NODISCARD void* operator new[](std::size_t) _GLIBCXX_THROW (std::bad_alloc)
       >       |                          ^
       > In function 'memset',
       >     inlined from 'Finalize' at /build/source/src/Cafe/OS/libs/nn_boss/nn_boss.cpp:1164:0,
       >     inlined from 'dtor' at /build/source/src/Cafe/OS/libs/nn_boss/nn_boss.cpp:1124:0,
       >     inlined from 'dtor' at /build/source/src/Cafe/OS/libs/nn_boss/nn_boss.cpp:1360:0,
       >     inlined from 'operator()' at /build/source/src/Cafe/OS/libs/nn_boss/nn_boss.cpp:1613:0,
       >     inlined from '_FUN' at /build/source/src/Cafe/OS/libs/nn_boss/nn_boss.cpp:1613:0:
       > /nix/store/932dj5qwfzck90mnvqpd1f9hjqznaqdj-glibc-2.40-36-dev/include/bits/string_fortified.h:59:33: warning: '__builtin_memset' writing 40 bytes into a region of size 0 overflows the destination [-Wstringop-overflow=]
       >    59 |   return __builtin___memset_chk (__dest, __ch, __len,
       >       |                                 ^
       > In function '_FUN':
       > lto1: note: destination object is likely at address zero
       > /nix/store/pbixrqd120nnw8a0ygfpm3mnwlr1mbj7-binutils-2.43.1/bin/ld: /build/ccqNZICu.ltrans71.ltrans.o: undefined reference to symbol '_ZN7glslang7TShader10setStringsEPKPKci'
       > /nix/store/pbixrqd120nnw8a0ygfpm3mnwlr1mbj7-binutils-2.43.1/bin/ld: /nix/store/09465jfdjm4lkddh5w1ih6q95vbmpmjh-glslang-15.0.0/lib/libglslang.so.15: error adding symbols: DSO missing from command line
       > collect2: error: ld returned 1 exit status
       > ninja: build stopped: subcommand failed.
```

### Notify maintainers
@zhaofengli
@baduhai
@AndersonTorres
### Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.58, NixOS, 24.11 (Vicuna), 24.11.20241029.807e915`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.9`
 - channels(root): `"nixos-23.11"`
 - nixpkgs: `/nix/store/m68ikm8045fj7ys7qvgr608z9l70hh1k-source`

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by AndersonTorres** [comment]
What is the version of cemu you are trying to build?

**Comment by joygnu** [comment]
2.0-92

**Comment by baduhai** [comment]
Duplicate of #354078, and already fixed by #353870.
